standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -2955,6 +2955,7 @@ PartitionsResponse get_partitions_req(1:PartitionsRequest req)
   void mark_cleaned(1:CompactionInfoStruct cr) throws(1:MetaException o1)
   void mark_compacted(1: CompactionInfoStruct cr) throws(1:MetaException o1)
   void mark_failed(1: CompactionInfoStruct cr) throws(1:MetaException o1)
+  void update_properties(1: CompactionInfoStruct cr) throws(1:MetaException o1)
```

Comment:
could we have better names for these methods - these function are present on the "hive metastore api"

and these methods doesn't really reflect that they only have to do with compaction

Reply:
changed to update_compaction_properties()